doc/README.md

```diff
@@ -0,0 +1,67 @@
+## Adding GRPC Support to Tailcall Overview
+
+This proposal outlines an approach for adding GRPC support to Tailcall to allow generating GRPC server stubs from GraphQL schemas.
+
+
+# Design Goals
+
+The primary design goals are:
+
+Allow configuring GRPC servers alongside HTTP servers in Tailcall config
+Generate GRPC server stubs from GraphQL schemas
+Enable handling GRPC requests with Tailcall resolvers
+Integrate cleanly with existing Tailcall architecture
+Proposed Solution
+The proposed solution is to:
+
+Add a grpc section to the Tailcall config file, similar to http:
+
+```bash
+grpc:
+  port: 50051
```

Comment:
- [ ] Where is this config written? We need to place it as an operator somewhere?
- [ ] How are multiple GRPC endpoints handled?

Reply:
>   [ ]  How are multiple GRPC endpoints handled?
We can handle multiple GRPC endpoints by defining multiple services in the .proto files from the GraphQL schema